src/app/public/modules/sectioned-form/sectioned-form.component.spec.ts

```diff
@@ -58,14 +66,15 @@ describe('Sectioned form component', () => {
   });
 
   function createTestComponent() {
```

Comment:
Any thoughts on if we can add a test or two around sectioned form? Maybe not since we aren't really changing anything other than passing the new flag into the service but I wanted to ask and see if you had any thoughts.

Reply:
I'm adding a similar test to the sectioned form that checks the order of content elements to make sure the order changes when `maintainTabContent == false`, but doesn't change when `maintainTabContent == true`